opal/mca/smsc/base/base.h

```diff
@@ -1,6 +1,9 @@
 /* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
 /*
- * Copyright (c) 2021      Google, LLC. All rights reserved.
+ *
+extern mca_smsc_module_t *selected_module;
```

Comment:
This looks like an errant paste or something...?

Reply:
Should be fixed now.